python_scripts/comp_to_prev_version.py

```diff
@@ -138,7 +138,7 @@ def plot_previous_results(observable, setup, filename, color_list = [], pdg = 0,
                 plt.plot(data['y'], data[str(energy)], color = plot_color,
                          linestyle = '-', linewidth = 10, zorder = 1, alpha = 0.15)
             elif 'ptspectra' in filename:
-                plt.plot(data['pt'], data[str(energy)], color = plot_color,
```

Comment:
Third order importance comment: you might consider to format the lines as int the case of 'mtspectra' above. It does not change anything in the functionality of the code, but it makes slightly faster to notice that the code does the same thing.

Reply:
Thank you, that is a very good point and I addressed this in the latest commit!